sssom/util.py

```diff
@@ -1440,7 +1440,7 @@ def get_all_prefixes(msdf: MappingSetDataFrame) -> list:
                         # print(
                         #     f"Slot '{slot}' has an incorrect value: {msdf.metadata[slot]}"
                         # )
-                        raise ValidationError(
+                        logging.warning(
```

Comment:
Why was this necessary?

Reply:
This does not seem the right way to solve this though..  Which process adds an `object_source` that is not a valid CURIE?

Reply:
[test_conversion.py](https://github.com/mapping-commons/sssom-py/blob/h2-patch3/tests/test_conversion.py#L106) <= [writers.py](https://github.com/mapping-commons/sssom-py/blob/h2-patch3/sssom/writers.py#L107) <= [validators.py](https://github.com/mapping-commons/sssom-py/blob/h2-patch3/sssom/validators.py#L73)

Reply:
None of this _adds_ an invalid object_source though that would justify this change in this PR now. 

Some observations (partially related):

1. OBJECT_SOURCE_ID does _not_ exist in SSSOM: https://github.com/mapping-commons/sssom-py/blob/master/sssom/parsers.py#L541
2. The URI parameter should validate correctly as an object_source: https://github.com/mapping-commons/sssom-py/blob/master/sssom/parsers.py#L543

Can you say what exactly the error is saying?

Reply:
https://github.com/mapping-commons/sssom-py/actions/runs/5137740141/jobs/9246227909

Reply:
Got it! Proceed then, its not the "validate" methods, its something else. Thanks!